aikau/src/main/resources/alfresco/renderers/Actions.js

```diff
@@ -131,6 +142,69 @@ define(["dojo/_base/declare",
       onlyShowOnHover: false,
 
       /**
+       * The label to display to as the actions renderer.
+       * 
+       * @instance
+       * @type {string}
+       * @default
+       * @since 1.0.46
+       */
+      label: "alf.renderers.Actions.menuLabel",
+
+      /**
+       * This will hold a reference to a diijt/Menu widget that is created to hold the menu. This
+       * should not be referenced by extending widgets as it may not always be available depending 
+       * upon how implementation of the menu changes as required. For example, this widget previously
+       * used an [AlfMenuBar]{@link module:alfresco/menus/AlfMenuBar} but it was necessary to change
+       * the implementation to resolve menu pop-up location issues (see AKU-706).
+       * 
+       * @instance
+       * @type {object}
+       * @default
+       * @since 1.0.46
+       */
+      _menu: null,
+
+      /**
+       * [destroy description]
+       * @param  {[type]} preserveDom [description]
+       * @return {[type]}             [description]
+       */
+      destroy: function alfresco_renderers_Actions__destroy(preserveDom) {
+         this.inherited(arguments);
+         if (this._menu)
+         {
+            this._menu.destroyRecursive();
+         }
+      },
+
+      /**
+       * Handles key press events on the actions renderer and opens the menu if 
+       * 
+       * @instance
+       * @param  {object} evt The key press event
+       * @since 1.0.46
+       */
+      onKeyPress: function alfresco_renderers_Actions__onKeyPress(evt) {
+         if (evt && (evt.charOrCode === keys.ENTER || evt.charCode === keys.SPACE))
```

Comment:
During development I observed that charOrCode can reliably be used for ENTER but not for SPACE.... this is why they are using different attributes. I think this is because SPACE has an actual character (i.e. " ") but ENTER doesn't.